### PR TITLE
Release 0.35.1

### PR DESCRIPTION
**Pull request by bcardiff**
Preview at https://github.com/bcardiff/crystal/blob/changelog/0.35.1/CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
+# 0.35.1 (2020-06-??)
```
Comment (Sija) [inline CHANGELOG.md:1]: Should have a full date of release :)

Comment (bcardiff) [reply CHANGELOG.md:1]: 🤦

Comment (jhass) [reply CHANGELOG.md:1]: We'll just  write it as "the mystery release" into Crystal's history books :D

Comment (straight-shoota) [reply CHANGELOG.md:1]: Should we add a CI job for validating changelog dates... 🤔 😆

**Pull request merged**